Source/LightboxTransition.swift

```diff
@@ -0,0 +1,76 @@
+import UIKit
+
+class LightboxTransition: NSObject {
```

Comment:
Yes, for some reason `UIViewControllerAnimatedTransitioning` needs NSObject, if I delete it it says that it doesn't conform to the protocol...